playbooks/roles/shadowsocks/tasks/main.yml

```diff
@@ -91,4 +91,4 @@
     path: "{{ shadowsocks_password_file }}"
     owner: root
     group: root
-    mode: 0600
+mode: 0600
```

Comment:
The indentation is off here. I don't think any changes to this file should be necessary.